index.html

```diff
@@ -301,6 +301,22 @@ <h3>
           </table>
         </section>
 
+        <section id="normative">
+          <h3>
+            Registries
+          </h3>
+          <p>
+The Working Group may create a set of
+<a href="https://www.w3.org/2021/Process-20211102/#registries">registries</a>
+including <a href="https://www.w3.org/2021/Process-20211102/#registry-definition">registry definitions</a>
+and <a href="https://www.w3.org/2021/Process-20211102/#registry-table">registry tables</a>
+to support extension points in the above normative deliverables. Registries for
```

Comment:
(Purely editorial) It may be that my non-native English is the problem, but the formulation sounds a bit odd to my ears... 
```suggestion
to support extension points in the normative deliverables listed in the previous sections. Registries for
```